packages/linux/package.mk

```diff
@@ -15,6 +15,8 @@ PKG_STAMP="$KERNEL_TARGET $KERNEL_MAKE_EXTRACMD $KERNEL_UBOOT_EXTRA_TARGET"
 
 PKG_PATCH_DIRS="$LINUX"
 
+[[ "$KERNEL_TARGET" == uImage* ]] && PKG_DEPENDS_TARGET+=" u-boot"
```

Comment:
Single `=` please! :)

Also single `[ ]` would suffice here too.

Reply:
OK for single `=` but would single `[ ]` work when there's a wildcard?

Reply:
Yep you're right, keep `[[ ]]` and only drop one `=` - thanks!

Reply:
Done.